Update Team

Pratik2026 opened the pull request:
This PR updates the team page by adding information about new members and sorts the team into different batches.

[tpc.webm](https://github.com/BitByte-TPC/TPC-website/assets/128511266/87d70bf3-6e99-4a24-86dc-719b6f445b89)


- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] The title of my pull request is a short description of the requested changes.

Pratik2026 committed:
feat(frontend): Added 2022 batch members in Team

Samy-33 commented on frontend/src/pages/Domains/index.tsx:
```diff
@@ -3,7 +3,7 @@ import { createStyles, makeStyles, Theme } from "@material-ui/core/styles";
 import Nav from "src/components/Nav";
 import DomainCard from "src/components/Cards/DomainCard";
-import { teamList } from "./teamList";
+import { teamList_22, teamList_21, teamList_20 } from "./teamList";
```
Instead of defining multiple lists for batches, can we have a field in the user object that specifies the batch?
That will be much more clear and easy to maintain.

Pratik2026 replied:
I have done the changes as requested. Kindly review it and let me know if any further adjustment is required

Pratik2026 committed:
refactor(frontend):added a batch field in teamlist and modified the code accordingly

Samy-33 commented on frontend/src/pages/Domains/index.tsx:
```diff
@@ -79,26 +90,96 @@ const Domains: React.FC = () => {
         <Typography className={classes.title} align="center">
           DEVELOPERS
         </Typography>
+        <Typography className={classes.subtitle} align="left">
+          2020 Batch
+        </Typography>
+        <div className={classes.flexbox}>
+          {teamList.map((memberData, i) => {
+            return memberData.team === "dev" && memberData.batch === 2020 ? (
+              <DomainCard key={i} {...{ ...memberData, team: "Developer" }} />
+            ) : null;
+          })}
+        </div>
```
This could be a separate component. If not a component then can loop over each batch's data.

Aksh-Bansal-dev replied:
Yes. We should do it in a way such that we don't have to make changes here for every batch.

As @Samy-33 mentioned, one way is to extract all the batches from the list and loop over them.

Samy-33 commented:
@Aksh-Bansal-dev, can you take a look at this?

Aksh-Bansal-dev reviewed (commented):
Please attach screenshots as well.

Pratik2026 committed:
refactor(frontend): Created a TeamsCard component.

Adds a TeamCard component which takes batch and domain
as props to display Team-Member details accordingly.

Pratik2026 commented:
I have created a separate component as suggested. This component is taking batch and domain as props to display the team-member cards accordingly.

Aksh-Bansal-dev reviewed (commented):
Compress the large images (>1MB) otherwise the page load times will be high.

Aksh-Bansal-dev reviewed (approved):
LGTM. 
Thanks for contributing!

Pratik2026 committed:
update(frontend): Add some 2021 batch members in team.

Pratik2026 committed:
refactor(frontend): compress the large image to enhance the loading
speed.
